docs/use-linea/explore/use-patch.md

```diff
@@ -4,9 +4,10 @@ sidebar_position: 1
 ---
 
 # Patch Wallet
+
 [Patch Wallet](https://linea.patchwallet.com/) is a "zero onboarding" wallet attached to every Twitter account, email, and telephone number. There's no app to download, no gas, or even sign-up. Every Twitter user has a wallet automatically and can start receiving tokens and NFTs!
 
-In this guide, we'll walk you through how to access your Patch wallet connected to your Twitter, mint an NFT, and send it to *any* Twitter user...[Elon Musk](https://linea.patchwallet.com/twitter:elonmusk), [Cristiano Ronaldo](https://linea.patchwallet.com/twitter:cristiano), [Narendra Modi](https://linea.patchwallet.com/twitter:narendramodi) or anyone else!
+In this guide, we'll walk you through how to access your Patch wallet connected to your Twitter, mint an NFT, and send it to _any_ Twitter user...[Elon Musk](https://linea.patchwallet.com/twitter:elonmusk), [Cristiano Ronaldo](https://linea.patchwallet.com/twitter:cristiano), [Narendra Modi](https://linea.patchwallet.com/twitter:narendramodi) or anyone else!
```

Comment:
I think the ellipsis here needs to be a dash + space

Reply:
will change.